web/src/utils/formatting.ts

```diff
@@ -15,7 +15,7 @@ export const formatPower = function (d: number, numberDigits = DEFAULT_NUM_DIGIT
   return power;
 };
 
-export const formatCo2 = function (d, numberDigits = DEFAULT_NUM_DIGITS) {
+export const formatCo2 = function (d: number, numberDigits = DEFAULT_NUM_DIGITS) {
```

Comment:
```suggestion
export const formatCo2 = function (d: number, numberDigits: number = DEFAULT_NUM_DIGITS) {
```
While it can implicitly infer that this is a number marking it as one will ensure DEAFULT_NUM_DIGITS is always a number and error if it changes for whatever reason.

Reply:
Good point!